### PR TITLE
fix management plan form history stack

### PR DESCRIPTION
**Pull request by JimmyRowland**

### CODE REVIEW

File: packages/api/src/controllers/taskController.js [hunk 1]
```diff
@@ -106,8 +106,8 @@ const taskController = {
           happiness,
           duration,
           wage_at_moment: override_hourly_wage ? wage_at_moment : wage.amount,
-        });
-        return result ? res.sendStatus(200) : res.status(404).send('Task not found');
+        }).returning('*');
+        return result ? res.status(200).send(result) : res.status(404).send('Task not found');
```
Comment (smattingly) [inline packages/api/src/controllers/taskController.js:110]: Should respond with 200 and empty array.

**Review by smattingly — CHANGES_REQUESTED** [review]
Small change: 404 -> 200

File: packages/api/src/controllers/taskController.js (hunk 1 above)
Comment (JimmyRowland) [reply packages/api/src/controllers/taskController.js:110]: It's a patch endpoint.

Comment (smattingly) [reply packages/api/src/controllers/taskController.js:110]: Oh, I see. In that case I wonder if it is safe to assume that a falsy `result` is necessarily due to not finding the task? But I'll leave that up to you.

**Review by smattingly — APPROVED** [review]

**Pull request merged**